src/dualscreen.tsx

```diff
@@ -27,19 +27,20 @@ import {ImagePlayer, SlideItem, SlideThumbnail} from './image-player';
 import {core, ui, BasePlugin, IEngineDecoratorProvider, KalturaPlayer} from '@playkit-js/kaltura-player-js';
 import {OnClickEvent} from '@playkit-js/common/dist/hoc/a11y-wrapper';
 import './styles/global.scss';
+import { DualscreenEvents } from "./events";
 
 const {
   reducers: {shell}
 } = ui;
-const {EventType} = core;
+const {EventType, FakeEvent} = core;
 
 const HAS_DUAL_SCREEN_PLUGIN_OVERLAY = 'has-dual-screen-plugin-overlay';
 const PRESETS = ['Playback', 'Live', 'Ads'];
 const IMAGE_PLAYER_ID = 'imagePlayer';
 const MAIN_PLAYER_ID = 'mainPlayer';
 
 export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineDecoratorProvider {
-  private _layout: Layout;
+  private layout!: Layout;
```

Comment:
question = why do you need the "!", since layout can be null/

Reply:
it can’t be null, it is assigned on the constructor, so it is never = null